src/hidimstat/distilled_conditional_randomization_test.py

```diff
@@ -610,69 +567,41 @@ def _joblib_distill(
     return X_residual, sigma2, y_residual
 
 
-def _fit_lasso(
+def run_lasso_screening(
     X,
     y,
-    n_jobs,
-    alpha,
-    alphas,
-    n_alphas,
-    alpha_max_fraction,
-    **params,
+    lasso_model=LassoCV(fit_intercept=False),
```

Comment:
```suggestion
    lasso_model=LassoCV(fit_intercept=False, random_state=0),
```

For default parameters, I prefer that the random_state is fixed.

Reply:
No, Lasso is also a source of randomness. 
Lasso in scikitlearn requires a random generator as a parameter.

Reply:
In your actual modification, I count 5 random generators to define for having reproducible results:
- 2 for the cross-validation generator of the LassoCV
- 2 for the model of lasso
- 1 for the method

For the simplicity of handling the random generator, I prefer that all the other random generators as a fixed seed except the random generator of the method.

Reply:
Yes, but the generator of the method is actually never used. I suggest fixing this in #368 
Consequently, if you fix the seed of the models, there is no more randomness by default, which I think should be the default behavior. 

Let me know what you think.

Reply:
OK